lxd/storage_lvm_utils.go

```diff
@@ -842,30 +860,51 @@ func lvmCreateLv(projectName, vgName string, thinPoolName string, lvName string,
 	lvSizeInt = int64(lvSizeInt/512) * 512
 	lvSizeString := units.GetByteSizeString(lvSizeInt, 0)
 
+	numStripesString := fmt.Sprintf("%d", numStripes)
+	stripeSizeUint, err := units.ParseByteSizeString(stripeSize)
+
+	if err != nil {
+		return err
+	}
+
+	stripeSize = units.GetByteSizeString(stripeSizeUint, 0)
+	stripesSizeString, err := getLVCreateSize(stripeSize)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Why all of this logic? Can't you just use the value from `ParseByteSizeString` and pass that as bytes (`B` suffix) to lvcreate?